spec/requests/register_api/get_multiple_applications_spec.rb

```diff
@@ -43,6 +43,25 @@
     expect(parsed_response).to be_valid_against_openapi_schema('MultipleApplicationsResponse')
   end
 
+  it 'returns paginated results if the total exceeds page size' do
+    create_list(
+      :application_choice,
+      5,
+      :with_recruited,
+      :with_course_uuid,
```

Comment:
I introduced this trait on the `application_choice` factory because (a) `course_option: create(:course_option, :with_course_uuid)` doesn't work because you can't share the same course option with multiple application choices and (b) Rubocop wouldn't let me just put this in a `5.times` loop.